lib/Interpreter/CppInterOp.cpp

```diff
@@ -1284,12 +1284,22 @@ namespace Cpp {
 #endif
       }
       auto VDAorErr = compat::getSymbolAddress(I, StringRef(mangledName));
-      if (!VDAorErr) {
-        llvm::logAllUnhandledErrors(VDAorErr.takeError(), llvm::errs(),
-                                    "Failed to GetVariableOffset:");
-        return 0;
+      if (VDAorErr)
+        return (intptr_t)jitTargetAddressToPointer<void*>(VDAorErr.get());
+
+      llvm::logAllUnhandledErrors(VDAorErr.takeError(), llvm::errs(),
+                                  "Failed to GetVariableOffset:");
+
+      if (VD->isConstexpr() && !VD->hasInit())
```

Comment:
Shouldn’t here the check be if the VD is a template, implicitly instantiated with no init?

Looks like we don’t have test coverage for this case though…